Ship_Game/GameScreens/MainMenu/MainMenuScreen.cs

```diff
@@ -116,11 +116,9 @@ void CreateVersionArea()
                 VersionArea.Add(new VersionLabel(this, offset, ScreenHeight - 38, modTitle) { Name = "mod_title" });
         }
 
-        public void ResetMusic()
+        protected override void ResetMusic()
         {
-            GameAudio.ConfigureAudioSettings(GlobalStats.MusicVolume, GlobalStats.EffectsVolume);
-            GameAudio.StopGenericMusic(fadeout: false);
-            ScreenManager.Music.Stop();
+            base.ResetMusic();

```

Comment:
```cs
ScreenManager.StopMusic();
```
We should get rid of `CustomMenuMusic` and instead of SD_Theme_Reprise_06 we could create a `MainMenuMusic` entry in AudioConfig.yaml

The "TitleTheme" and "Female_02_loop" are actually "VictoryMusic" and "DefeatMusic"

Reply:
The "TitleTheme" was originally used in YouWinScreen.cs
And "Female_02_loop" was originally used in YouLoseScreen.cs

Reply:
done